package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vscode-choosealicense",
   "displayName": "Choose a License",
-  "version": "0.9.2",
+  "version": "0.10.0",
```

Comment:
Please revert this, I will bump on release.

Reply:
OK, I got it.